src/repos/ArticleRepo.ts

```diff
@@ -60,17 +60,12 @@ export const compareTrendiness = (a1: Article, a2: Article) => {
  *
  * @function
  * @param {number} limit - number of articles to retrieve.
- * @param {string} since - retrieve articles after this date.
  */
 const getTrendingArticles = async (
-  since: string,
   limit = Constants.DEFAULT_LIMIT,
 ): Promise<Article[]> => {
-  const articlesSinceDate = await ArticleModel.find({
-    date: { $gte: new Date(new Date(since).setHours(0, 0, 0)) },
-  }).exec();
-
-  return articlesSinceDate.sort(compareTrendiness).slice(0, limit);
+  const articles = await ArticleModel.find({}).exec();
```

Comment:
Just curious as to why we need to find and then exec, does await not get the articles? 

I guess a better question is what does `exec()` do?

Reply:
https://mongoosejs.com/docs/promises.html#should-you-use-exec-with-await? that article should explain it, but i realized idk if all queries use .exec() so I'll review to make them consistent

Reply:
TLDR, .exec() just gives a better stack trace